### PR TITLE
[v3r1p2] allow to configure backround and logo images

### PR DESCRIPTION
**Pull request by zmathe**

### CODE REVIEW

File: WebApp/handler/RootHandler.py [hunk 1]
```diff
@@ -90,11 +90,15 @@ def web_index(self):
       open_app = xss_filter( self.request.arguments["open_app"][0].strip() )
 
     icon = data[ 'baseURL' ] + Conf.getIcon()
-
+    backround = data[ 'baseURL' ] + Conf.getBackround()
```
Comment (fstagni) [inline WebApp/handler/RootHandler.py]: `background` not `backround` :)

File: Lib/Conf.py [hunk 2]
```diff
@@ -136,3 +136,9 @@ def SSLProrocol():
 
 def getStaticDirs():
   return getCSValue( "StaticDirs", [] )
+
+def getLogo():
+  return getCSValue("Logo","/static/core/img/icons/system/_logo_waiting.gif")
+
+def getBackround():
```
Comment (fstagni) [inline Lib/Conf.py]: `getBackgroud`

File: WebApp/template/root.tpl [hunk 3]
```diff
@@ -82,6 +82,7 @@
           GLOBAL.VALID_VIEWS = ["desktop","tabs"];
           GLOBAL.MAIN_VIEW_SAVE_STRUCTURE_VERSION = 1;
           GLOBAL.OPEN_APP = "{{open_app}}";
+          GLOBAL.BACKGROUND = "{{backroundImage}}";
```
Comment (fstagni) [inline WebApp/template/root.tpl]: and here.

File: WebApp/template/root.tpl [hunk 4]
```diff
@@ -113,6 +114,7 @@
...
```
Comment (fstagni) [inline WebApp/template/root.tpl]: and here

File: Lib/Conf.py (hunk 2 above)
Comment (zmathe) [reply Lib/Conf.py]: Done

File: WebApp/static/core/js/views/tabs/RightContainer.js [hunk 5]
```diff
@@ -416,7 +416,7 @@ Ext.define('Ext.dirac.views.tabs.RightContainer', {
                   isLoaded : isLoaded,
                   bodyStyle : {
                     background : '#AAAAAA',
-                    backgroundImage : 'url(/DIRAC/static/core/img/wallpapers/dirac_background_6.png)',
+                    backgroundImage : 'url('+GLOBAL.BACKGROUND+')',
```
Comment (fstagni) [inline WebApp/static/core/js/views/tabs/RightContainer.js]: This is mostly my ignorance with js, so can you explain why at this line you used `+GLOBAL.BACKGROUND+` and at line 83 only `GLOBAL.BACKGROUND` ?

Comment (zmathe) [reply WebApp/static/core/js/views/tabs/RightContainer.js]: In the line 83 a setter method is used. The wallpaper is a widget...  The setter method is doing all required rendering, etc. 
+GLOBAL.BACKGROUND+ is used inside a custom CSS property (bodyStyle) as you can see in line 417.  I set the background using CSS element. You can see how you can set a background using CSS: https://www.w3schools.com/cssref/pr_background-image.asp

**Pull request merged**